spinoso-string/src/lib.rs

```diff
@@ -1666,6 +1668,25 @@ impl String {
         inner(self.inner.as_slice(), needle, offset)
     }
 
+    /// Returns the index of the last occurrence of the given substring in this
+    /// `String`.
+    ///
+    /// Returns [`None`] if not found. If the second parameter is present, it
+    /// specifies the position in the string to begin the search.
```

Comment:
Is it "byte position" or "character position"?

If it is byte index, can you add a test (not a doc test, a real one) where the position is in the middle of a multi byte UTF-8 character?

Reply:
> Is it "byte position" or "character position"?

fixed in d8eda7b7fe5eba53c460b4d80c28694da2905fe6

> If it is byte index, can you add a test (not a doc test, a real one) where the position is in the middle of a multi byte UTF-8 character?

Sure. I test this as part of `byteindex_support_specifiying_byte_position_to_start_search`, added in 0351cb2.